esim_go_client.py

```diff
@@ -1,28 +1,88 @@
-import urllib3
+import requests
```

Comment:
https://stackoverflow.com/questions/40741282/cannot-use-requests-module-on-aws-lambda you might need to follow this to get `requests` working

Reply:
@zaine2003 we can use urllib3